docs/getting-started-with-scalardb.md

```diff
@@ -130,6 +135,14 @@ public class ElectronicMoneyWithStorage extends ElectronicMoney {
 }
 ```
 
+Note that you need to use `JdbcDatabaseConfig` when you use JDBC databases as follows.
+
+```java
+    Injector injector = Guice.createInjector(new StorageModule(new JdbcDatabaseConfig(props)));
```

Comment:
It's not a question to the doc, but do we still need to pass `JdbcDatabaseConfig` for StorageModule?
I think we updated to create `JdbcDatabaseConfig` in the module  for transaction.